README.md

```diff
@@ -35,7 +35,7 @@ Please follow [this section in original template](https://github.com/fredrikhgre
 All software is provided and run with docker. See the [Makefile](Makefile) for inspiration.
 
 ## Usage
-The following command will run nifi registry in the [example/standalone_git](example/standalone) folder.
+The following command will run nifi registry in the [example/standalone_git](example/standalone_git) folder. (__NOTE!__ Please read [README.md](example/standalone_git/README.md) before executing the example)
```

Comment:
could you please give  a link in readme which file should be set before run make up ?

Reply:
could you also write what should be the git_access_user =" " forexample :users git _access_password ="token"

Reply:
> could you give a link in readme which file should be set before run make up ?

There exists a link in [README.md](https://github.com/hannemariavister/terraform-nomad-nifiregistry/blob/15ab80d765fa1fadf3b1d1cef651996688f9714f/README.md#usage) in section `Usage`. Is that clear enough ?

Reply:
thats enough

Reply:
> could you also write what should be the git_access_user =" " forexample :users git _access_password ="token"

I don’t think `git_access_password` is limited to only accept `token`. You can authorise using volume (see example here: https://github.com/michalklempa/docker-nifi-registry#cloning-using-gitssh ). So I am not sure we should write that..